README.md

```diff
@@ -20,7 +20,6 @@ defargs @arg-session {
 # Installation
 
 * TODO -- Nix derivation
-* TODO -- fish completion (port from ruby tabry)
```

Comment:
Sounds good, I'll leave it in... By "directory" special, do you just mean it should only complete for directories vs all files? Also, what do you mean by "delegate" special?

Reply:
Correct about the 'directory' special.
The 'delegate' special is something I added recently (and hope to approve soon), it delegates to another command's completion. for instance

```
cmd dcr
arg {
  opts delegate "docker compose run"
}
```

Reply:
Oh, I think I see now... Can you describe what the output from tabry looks like for delegates? Right now, tabry_fish only handles output from tabry that looks like:

```
foo
bar

file
dir
```

Reply:
Try putting this in `~/.local/share/tabry/dcr.tabry`:

```
cmd dcr
sub foo
arg { opts delegate "docker compose run" }
```

if you run `tabry "dcr " 4` the output will be:

```
foo

delegate docker compose run
```

Basically tabry returns normal completions + blank line + special completions (handled by the shell script). The "delegate" line will be handled by the shell script.

It may be tricky to implement so I think it's not a mandatory feature for now.
Also, in the future I want to support a "varargs delegate" type where it will start delegating the current arg and all follwing args to the delegated completions.

Reply:
Sounds good, I'll add delegate support in another PR!